.pa11yci

```diff
@@ -0,0 +1,25 @@
+{
+  "defaults": {
+    "timeout": 10000
+  },
+  "urls": [
```

Comment:
Could also add `http://localhost:4200/login` and `http://localhost:4200/register`

Reply:
Added